tests/README.md

```diff
@@ -14,11 +14,12 @@ As best practice, please also include any source used to make the swf - such as
 
 # Test Structure
 ## test.toml
-Except for `num_frames`, every other field and section is optional.
+Except for `num_ticks`, every other field and section is optional.
 
 ```toml
-num_frames = 1 # The amount of frames of the swf to run
-sleep_to_meet_frame_rate = false # If true, slow the tick rate to match the movies requested fps rate
+num_ticks = 1 # The amount of frames of the SWF to run.
```

Comment:
Did you rename `num_frames` to `num_ticks`? If so, why do tests still have `num_frames`?

Reply:
A few tests failed when I tried switching everything over to ticks so it's opt-in for now.

Reply:
Can you add `num_frames` back to the documentation and explain that it's one or the other then? I don't want the docs to not match what we're actually using

Reply:
Sure, but we ideally want all new tests to be in ticks, not frames. So I mentioned that `num_frames` is a deprecated option.